sycl/doc/extensions/experimental/sycl_ext_oneapi_non_uniform_groups.asciidoc

```diff
@@ -230,14 +223,12 @@ groups is discouraged, since it is highly likely that such code would have to
 make assumptions regarding work-item scheduling and forward progress
 guarantees.
 
-The following group functions support the `ballot_group`, `fixed_size_group`,
-`tangle_group`, and `opportunistic_group` group types:
+The following group functions support user-defined groups:
```

Comment:
User-constructed instead of user-defined?

Reply:
Fixed in ebe85779e479e18f0776f270552c3796a7ade01b.